filebeat/input/unix/config.go

```diff
@@ -1,41 +0,0 @@
-// Licensed to Elasticsearch B.V. under one or more contributor
-// license agreements. See the NOTICE file distributed with
-// this work for additional information regarding copyright
-// ownership. Elasticsearch B.V. licenses this file to you under
-// the Apache License, Version 2.0 (the "License"); you may
-// not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
-
-package unix
-
-import (
-	"time"
-
-	"github.com/dustin/go-humanize"
-
-	"github.com/elastic/beats/v7/filebeat/inputsource/unix"
-)
-
-type config struct {
-	unix.Config `config:",inline"`
-}
-
-func defaultConfig() config {
-	return config{
-		Config: unix.Config{
-			Timeout:        time.Minute * 5,
-			MaxMessageSize: 20 * humanize.MiByte,
-			SocketType:     unix.StreamSocket,
-			LineDelimiter:  "\n",
-		},
-	}
-}
```

Comment:
Why was it moved to `filebeat/input/unix/input.go`?

Reply:
Code locality; having it in a separate file added no clarity.

Reply:
While I generally agree with not keeping the config on a separate file, every (as far as I know) input on Filebeat uses a `config.go` file to hold it's configuration struct and the `defaultConfig` function.

To keep the project consistent I believe it's better to keep this file untouched.

Reply:
Please take a look at udp and tcp.

Reply:
also, I would like to preserve the blame/changelog on the config-related file. Moving it to another file would discard all of that.